BUILD.md

```diff
@@ -12,7 +12,6 @@
     - To build on Windows, see the [Building on Windows](#building-on-windows) section. _MSVC (Visual Studio) is recommended._
 4. `sudo make install`
     - Default install location is `/usr/local`
-    - On Debian/Ubuntu, instead of installing files directly with `sudo make install`, you can run `cd build && cpack -G DEB && sudo dpkg -i nvim-linux64.deb` to build DEB-package and install it. This should help ensuring the clean removal of installed files.
```

Comment:
Why not mention this?

Reply:
I thought we concluded deb packages was ultracringe because it doesn't really have any granularity with other dependencies and that it's just a shitty container.

Reply:
We don't provide a deb (for good reason), but mentioning `cpack -G DEB` is a nice hint for people that want to explore it. 

I don't remember the exact problems caused by the deb, did it leave the system in a bad state?

Reply:
Not really, from my conversations with ubuntu users this was never really what they wanted since it didn't update neovim when upgrading with apt. I don't think anyone really uses this, nor does there seem to be any good reason to honestly.

Reply:
Can re-add if want, I'm just going by gut feeling here.

Reply:
generally it's high-leverage when we can give a hint about an easy (one-liner) way to unlock powerful but non-obvious things (in this case, cmake generators are not typically known), but if it's not useful then can drop it.

Reply:
Makes sense. I still have a suspicion that this is a net negative because it will probably give incorrect hint for the majority of users that would be better served by using a package manager like apt/PPA, but are simply not aware of it. Let's remove it for the time being unless someone protests or it becomes relevant in the future.